config/alttp.php

```diff
@@ -2,6 +2,7 @@
 
 return [
     'base_rom' => env('ENEMIZER_BASE', null),
+    'api_throttle_whitelist' => explode(',', env('API_THROTTLE_WHITELIST', '')),
```

Comment:
just cause I'm being hazy, this doesn't get returned from any api endpoint, right? (I think we specifically call out keys, but just want to be sure)

Reply:
Yea, I was concerned about that too, but yes we call specific keys based on everything I see.  I can double-check this tonight though.

If all else fails I can move this to a new file too.

Reply:
Yea, we're not including the entire alttp key in any endpoint return, based on what I see when searching the code.  The `/randomizer/settings`, `/customizer/settings`, and `/base_rom/settings` don't show the api_throttle_whitelist key.  I determined this by doing a search for `config('alttp` and `config("alttp` in VSCode.

I strongly believe this is safe.